src/components/ae-button-group/ae-button-group.md

```diff
@@ -0,0 +1,43 @@
+```vue
+  <ae-button-group>
+    <ae-button>Button</ae-button>
+    <ae-button>Group</ae-button>
+  </ae-button-group>
+``` 
+
+### prop: face
+```vue
+  <ae-button-group>
+    <ae-button face="round" fill="primary">Button</ae-button>
+    <ae-button face="round" fill="primary">Group</ae-button>
+  </ae-button-group>
+``` 
+
+```vue
+  <ae-button-group>
+    <ae-button face="round" fill="secondary">Button</ae-button>
+    <ae-button face="round" fill="secondary">Group</ae-button>
+  </ae-button-group>
+``` 
+
+```vue
+  <ae-button-group shadow>
+    <ae-button face="round" fill="neutral">Button</ae-button>
+    <ae-button face="round" fill="neutral">Group</ae-button>
+  </ae-button-group>
+``` 
+
+```vue
+  <ae-button-group>
+    <ae-button face="round" fill="alternative">Button</ae-button>
+    <ae-button face="round" fill="alternative">Group</ae-button>
+  </ae-button-group>
```

Comment:
Documentation is not a tests, so I think only one example with ` <ae-button face="round">` is enough. All possible colors of AeButton are already included in the documentation of AeButton component.